nav2_bringup/bringup/launch/slam_launch.py

```diff
@@ -17,11 +17,14 @@
 from ament_index_python.packages import get_package_share_directory
```

Comment:
https://app.circleci.com/pipelines/github/ros-planning/navigation2/4753/workflows/14e3b0f6-9171-4b98-88ca-0aef32421b05/jobs/17651 bunches of linter errors

Reply:
Sorry, I was not too eager on giving access to my github account to a third party application just to see CI results. I now saw that there's an option to give access only to Public repos. I'll check now.